reporter/src/funTest/kotlin/reporters/EvaluatedModelReporterFunTest.kt

```diff
@@ -67,7 +59,7 @@ private fun generateReport(reporter: EvaluatedModelReporter, ortResult: OrtResul
     val input = ReporterInput(
         ortResult = ortResult,
         resolutionProvider = DefaultResolutionProvider().add(ortResult.getResolutions()),
-        howToFixTextProvider = HOW_TO_FIX_TEXT_PROVIDER
+        howToFixTextProvider = { "Some how to fix text".trimIndent() }
```

Comment:
Can't we remove the `.trimIndent()` here?

Reply:
Done.